de-lib/src/main/java/org/iplantc/de/teams/client/presenter/TeamsPresenterImpl.java

```diff
@@ -200,6 +211,46 @@ public void onTeamSaved(TeamSaved event) {
         });
     }
 
+    @Override
+    public void onLeaveTeamSelected(LeaveTeamSelected event) {
+        Group group = event.getGroup();
```

Comment:
Consider moving the `group` variable declaration inside the body of the `AsyncCallback#onSuccess` method.  It's good practice to minimize the scope of variables.